packages/ag-charts-website/e2e/toolbar.spec.ts

```diff
@@ -126,4 +126,46 @@ test.describe('toolbar', () => {
         await page.click(SELECTORS.canvas, { position: { x: 100, y: 100 } });
         await expect(page).toHaveScreenshot('callout-6-change-fill-color.png', { animations: 'disabled' });
     });
+
+    test('AG-13008 delete annotation', async ({ page }) => {
```

Comment:
Also, can we just put some simple tests around this into the existing `test('text', ...` block? These tests are supposed to be just smoke tests.

Reply:
I've added tests to `test('text', ...)` for the <kbd>Backspace</kbd> and <kbd>Delete</kbd> keys